packages/core/src/config/config.ts

```diff
@@ -56,6 +56,10 @@ export interface AccessibilitySettings {
   disableLoadingPhrases?: boolean;
 }
 
+export interface DisplaySettings {
```

Comment:
rather than placing this under config.ts in the core package please place this in settings.ts in the cli package.